src/LineChart.js

```diff
@@ -57,6 +58,7 @@ function LineChart({ data, parts, theme, ...props }) {
       <VictoryGroup>
         {groupData.map((gd, i) => (
           <VictoryGroup
+            key={JSON.stringify(gd)}
```

Comment:
`gd` is an array, right? `stringify`-ing it _just_ for `key` seems to be too much.

Reply:
@kilemensi I can't think of anything else unique here apart from index

Reply:
Cool, let’s observe the performance when used in Dominion or Takwimu.... If good enough, then no problem @karimkawambwa